### PR TITLE
Docker continuation

### PR DESCRIPTION
**Pull request by cordoval**

### CODE REVIEW

**Comment by cordoval** [comment]
@gushphp/contributor this can now push .gush.yml to env var GUSH_CONFIG and the loading of configuration will take place now. We cannot write back yet to configure gush but that is another problem.

**Comment by pierredup** [comment]
I don't think forcing everyone to use Gush through Docker is a good approach. It makes Gush harder to use, and harder to setup

**Comment by cordoval** [comment]
getting very close @sstok @gushphp/contributor 
```
~ docker run -it -v ~/.ssh:/root/.ssh -e GUSH_CONFIG=`cat ~/.gush/.gush.yml | base64` gush -vvvv p:create


  [RuntimeException]
  Bad owner or permissions on /root/.ssh/config
  fatal: Could not read from remote repository.
  Please make sure you have the correct access rights
  and the repository exists.
  error: Could not fetch gushphp


pull-request:create [--base BASE] [--source-org SOURCE-ORG] [--source-repo SOURCE-REPO] [--source-branch SOURCE-BRANCH] [--title TITLE] [-ra|--repo-adapter REPO-ADAPTER] [-o|--org ORG] [-r|--repo REPO] [-t|--template TEMPLATE]
```

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+chown -rf root:root ~/.ssh/config
+chmod -rf 644 ~/.ssh/config
```
Comment (pierredup) [inline start.sh]: Can't this be set in the Dockerfile, so that it is run once, and not each time you want to use Gush?

Comment (cordoval) [inline start.sh]: yes i am just troubleshooting before building it every time, it is my poor attempt to save time

Comment (cordoval) [inline start.sh]: can you help me here? i wonder how it is done in other projects, basically i am running into the permissions problems while mounting the ~/.ssh with proper keys

Comment (cordoval) [inline start.sh]: also it has to mount the keys so it is dynamic so that it can use your keys

**Comment by cordoval** [comment]
@pierredup now i get this
```
~ docker run -it -v ~/.ssh/id_rsa.pub:/root/.ssh/id_rsa.pub -v ~/.ssh/id_rsa:/root/.ssh/id_rsa -e GUSH_CONFIG=`cat ~/.gush/.gush.yml.bk | base64` gush -vvvv b:p -u origin
The authenticity of host 'github.com (192.30.253.112)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpXWGl7E1IGOCspRomTxdLUILviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
```

any thought on avoiding the confirmation?

**Comment by pierredup** [comment]
You can try `ssh-agent` to add the key, E.G

```
ADD ~/.ssh/mykey /tmp/  
RUN ssh-agent /tmp
```

**Comment by cordoval** [comment]
@gushphp/contributor @pierredup do you know why the error on circleci? it seems there is a prophesize method that is non existing? why in travis we did not see that before?

**Comment by cordoval** [comment]
bye bye travis 💃

**Pull request merged**